phone/src/apps/contacts/hooks/state.ts

```diff
@@ -17,6 +17,11 @@ export const contactsState = {
             undefined,
             buildRespObj(BrowserContactsState),
           );
+
+          if (resp.status !== 'ok') {
+            throw new Error(resp.errorMsg);
+          }
```

Comment:
This should probably be moved into "fetchNui" and throw there instead. But that's a larger refactor. Even if it's a simple one.